tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/run/JarRunFunctionPositiveTestCase.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.test.run;
+
+import org.ballerinalang.test.BaseTest;
+import org.ballerinalang.test.context.BallerinaTestException;
+import org.ballerinalang.test.context.LogLeecher;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * This class tests invoking main function in a bal for jvm target via Ballerina Build Command and run it the jar for
+ * testing given outputs.
+ */
+public class JarRunFunctionPositiveTestCase extends BaseTest {
+
+    private Path tempProjectDir;
+
+    private String jarPath = "src/test/resources/run/jar/test_main_for_jvm_target.jar";
```

Comment:
shall we not create the jar within the resource dir itself? shall we use the target directory to create the jar ?

Reply:
Fixed